### PR TITLE
add atompaw

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/atompaw/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX \
+   --with-linalg-libs="-L${PREFIX}/lib -lblas -llapack -lpthread" \
+   IFLAGS="-I${SRC_DIR}/include -I${PREFIX}/finclude" \
+   CC="${CC}" \
```
Comment (hmaarrfk) [inline recipes/atompaw/build.sh:6]: I guess these two lines are necessary

File: recipes/atompaw/build.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX \
+   --with-linalg-libs="-L${PREFIX}/lib -lblas -llapack -lpthread" \
+   CFLAGS="${CFLAGS} -L${PREFIX}/lib -llapack -lblas" \
```
Comment (hmaarrfk) [inline recipes/atompaw/build.sh:5]: are these two flags necessary?

File: recipes/atompaw/build.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX \
+   --with-linalg-libs="-L${PREFIX}/lib -lblas -llapack -lpthread"
```
Comment (isuruf) [inline recipes/atompaw/build.sh:4]: Can you enable shared?

Comment (jan-janssen) [reply recipes/atompaw/build.sh:4]: As far as I understand it is not supported by the developers https://github.com/atompaw/atompaw/blob/master/configure.ac#L173

Comment (isuruf) [reply recipes/atompaw/build.sh:4]: They are talking about inconvenience of not having a single executable, which is not something we have to worry about here.

Comment (jan-janssen) [reply recipes/atompaw/build.sh:4]: @isuruf I patched the configure script and updated the tests, can you take another look?

**Comment by jan-janssen** [comment]
@hmaarrfk and @isuruf Can you merge this PR ?

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/atompaw/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://users.wfu.edu/natalie/papers/pwpaw/{{ name }}-{{ version }}.tar.gz
+  sha256: 42a46c0569367c0b971fbc3dcaf5eaec7020bdff111022b6f320de9f11c41c2c
+  patches: 
+    - shared.patch
```
Comment (isuruf) [inline recipes/atompaw/meta.yaml:12]: You don't need to patch. You can do `./configure --enable-shared --disable-static`

Comment (jan-janssen) [reply recipes/atompaw/meta.yaml]: Ok, I did this

**Comment by jan-janssen** [comment]
@isuruf thx again for your suggestions, I updated the recipe accordingly. Is it ok to merge now?

**Pull request merged**